aldryn_newsblog/utils/utilities.py

```diff
@@ -109,20 +114,33 @@ def get_field_value(obj, name):
     return value
 
 
+def render_plugin(request, plugin_instance):
+    if getattr(plugin_instance, 'render_plugin', None):
+        # Backward compatibility for django-cms<3.4
+        return plugin_instance.render_plugin(
+            context=RequestContext(request),
+        )
+    else:
+        renderer = ContentRenderer(request)
+        context = RequestContext(request)
```

Comment:
On newer django versions (>= 1.9 I think), the RequestContext class will only apply context processors when binding to a template, it used to do it on __init__.

So in this case, you can just do `context = {'request': request}`.